docs/openshift_networking_requirements.md

```diff
@@ -0,0 +1,35 @@
+# OpenShift networking requirements
+Guidelines for a third party network plugin for OpenShift
+
+## Basic requirements
+OpenShift networking has certain requirements over and above kubernetes essentials. The basic kubernetes requirements can be found [here](https://github.com/kubernetes/kubernetes/blob/release-1.3/docs/design/networking.md).
+
+## CNI is the recommended way
+
+Any external networking solution can be used to plumb networking for openshift as long as it follows the 'CNI' spec. Then, openshift needs to be launched with 'networkPluginName: "cni"' in the master/node config yaml files. 
+
+When done through ansible, provide sdn_network_plugin_name=cni as the option while installing openshift. Be aware that openshift ansible installation allows a firewall passthrough for the VxLAN port (4789), so if a plugin needs other ports (for management/control/data) to be open, then the installer needs to be changed suitably.
```

Comment:
should mention that this goes in the inventory file.  Does it need to be in a specific group?  Example might also be helpful here.